data/phi0472/phi001/phi0472.phi001.perseus-eng4.xml

```diff
@@ -4,9 +4,9 @@ schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-
      <teiHeader>
           <fileDesc>
                <titleStmt>
-                    <title>Carmina</title>
+                    <title xml:lang="lat">Carmina</title>
                     <title type="sub">Machine readable text</title>
```

Comment:
I just noticed this — we've been removing the machine readable text. (There are just so many things....)